state/account.go

```diff
@@ -108,6 +108,17 @@ func (st *Account) Deserialize(buf []byte) error {
 	return nil
 }
 
+// SetNonce sets the nonce of the account
+func (st *Account) SetNonce(nonce uint64) error {
+	st.nonce = nonce
+	return nil
+}
+
+// PendingNonce returns the pending nonce of the account
+func (st *Account) PendingNonce() uint64 {
+	return st.nonce + 1
```

Comment:
for the new account, pending nonce should be = st.nonce? that is, if `st.nonce` = 0, we expect the next tx from this address carry `tx.nonce = 0`

Reply:
do you mean this commit https://github.com/iotexproject/iotex-core/pull/3387/commits/fb78e66897ef8c2270dbecefa23ede5b7d499f03? it is not part of this PR.

Reply:
ok, this will be changed in the final PR (new account type)